.github/workflows/pretest-and-test.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
     - name: Code Style
       run: |
-        pip install pysen black==21.11b1 flake8==4.0.1 isort==5.10.1 mypy==0.991
+        pip install pysen black==23.3.0 flake8==4.0.1 isort==5.10.1 mypy==0.991
```

Comment:
Change the version of black to eliminate problems caused by dependencies with click.